pkg/proxy/winuserspace/roundrobin.go

```diff
@@ -53,8 +53,8 @@ type affinityPolicy struct {
 
 // LoadBalancerRR is a round-robin load balancer.
 type LoadBalancerRR struct {
-	lock     sync.RWMutex
-	services map[proxy.ServicePortName]*balancerState
+	lock                    sync.RWMutex
+	winLoadbalancedServices map[proxy.ServicePortName]*balancerState
```

Comment:
I think old field name `services` is fine, "loadbalanced" is implied since the type is named `LoadBalancerRR`

Reply:
ok fair - i can move that back to what it was

Reply:
Done resolvinggg